lib/formats/gzip.js

```diff
@@ -6,23 +6,24 @@ const tar = require('tar-stream');
 module.exports = {
   label: 'gzip',
   extension: 'tgz',
-  'content-type': 'application/gzip',
+  type: 'application/x-compressed-tar',
```

Comment:
How can I change these values if needed?

Reply:
extension can be `tar.gz` and content-type be `application/gzip`.

A quick search show that some people had some issues with proxies/nginx/express corrupting the archive due to mismatching config between content-type and extension.